build.gradle

```diff
@@ -64,6 +69,14 @@ jar {
       'Built-Gradle': gradle.gradleVersion
     )
   }
+
+  // TODO: Remove this hack when plugin is fully compiled to Kotlin
+  from fileTree('/Users/noname/repo/gradle-versions-plugin/api/build/classes/kotlin/main/') {
```

Comment:
fyi, you can use `${projectDir}/api/...` or `project(':api').buildDir`, for the hack to work across multiple machines.